.github/workflows/check-docs.yml

```
@@ -0,0 +1,25 @@
+name: Check docs build
+# This workflow runs when a PR is labeled with `docs`
+# This will check if the docs build successfully by running `npm run build`
+on:
+  pull_request:
+    types: [ labeled ]
```

```suggestion
  paths:
      - './docs'
```

could do something like this to avoid needing to label